src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1272,12 +1302,16 @@ private List<Element<EmailNotificationDetails>> sendNotificationsToCitizenRespon
                 if (YesNoDontKnow.yes.equals(selectedRespondent.getValue().getDoTheyHaveLegalRepresentation())) {
                     log.info("Respondent is represented");
                     try {
-                        emailNotificationDetails.add(element(serviceOfApplicationEmailService.sendEmailNotificationToSolicitor(
-                            authorization, caseData,
-                            selectedRespondent.getValue(),
-                            isStaticDocs ? getNotificationPack(caseData, PrlAppsConstants.S, docs) : docs,
-                            SERVED_PARTY_RESPONDENT
-                        )));
+                        Map<String, Object> dynamicData = EmailUtils.getCommonSendgridDynamicTemplateData(caseData);
+                        dynamicData.put("name", selectedRespondent.getValue().getRepresentativeFullName());
```

Comment:
casename and reference already set in utils method

Reply:
addressed this Thank you !!